ui/src/overridden/backoffice/Importer/ImportedDocuments.js

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Accordion, Button, Header, Icon, Message } from 'semantic-ui-react';
+import _isEmpty from 'lodash/isEmpty';
+import { http } from '@inveniosoftware/react-invenio-app-ils';
+import _get from 'lodash/get';
+import { ReportDetails } from './ReportDetails';
+import { Link } from 'react-router-dom';
+import { BackOfficeRoutes } from '@inveniosoftware/react-invenio-app-ils';
+import { DocumentIcon } from '@inveniosoftware/react-invenio-app-ils';
+
+const importerURL = '/importer/check';
+
+export class ImportedDocuments extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeIndex: -1,
+      importCompleted: false,
+      data: null,
+      intervalId: null,
+      isLoading: true,
+    };
+  }
+
+  componentDidMount() {
+    //TODO: Define how much we wait until the next call (currently 30 sec)
+    const idVar = setInterval(() => this.checkForData(idVar), 30000);
+    // eslint-disable-next-line react/no-did-mount-set-state
+    this.setState({ intervalId: idVar });
+    this.checkForData(idVar);
+    window.addEventListener('beforeunload', this.onCloseWarning);
+  }
+
+  componentWillUnmount = () => {
+    const { intervalId } = this.state;
+    window.removeEventListener('beforeunload', this.onCloseWarning);
+    clearInterval(intervalId);
```

Comment:
Safer to check if defined:
```suggestion
    this.intervalId && clearInterval(this.intervalId);
```